modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
+
+.Prerequisites
+
+* DNS is configured via DHCP on the cluster.
```

Comment:
DHCP server must be available on the network we are connecting to. DNS may be configured there, but that is not important here